checador/app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,47 @@
+class EmployeesController < ApplicationController
+  
+  def index
+    @employees = Employee.all
+  end
+
+  def show
+    set_employee
+  end
+
```

Comment:
You can delete this too. if you don't need it, of course.

Reply:
Thank you, let me change it!:)